veps/sig-network/hotpluggable-nad-ref.md

```diff
@@ -0,0 +1,199 @@
+# VEP #140: Live Update NetworkAttachmentDefinition Reference
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [X] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
+- [ ] (R) Target version is explicitly mentioned and approved
+- [ ] (R) Graduation criteria filled
+
+## Overview
+
+VMs using secondary networks refer to a NetworkAttachmentDefinition object's namespace and/or name as part of their network specification. 
+Currently, the whole network specification, including this reference - is immutable for the lifecycle of a VM.
+If a user wishes to change the network a VM is connected to (example: switching VLANs), they must modify the VM specification and restart the VM. 
+This proposal introduces the ability to update the NAD reference (networkName) of a network on a running VM through Live Migration. 
+This allows the host network connection to change without any reboot.
+
+## Motivation
+
+VMs often need to move between logical network segments without service interruption. Some of the scenarios include:
+- moving VMs to a new VLAN
+- TBD
+
+## Goals
+
+- Change an existing NAD reference (`spec.networks[].multus.networkName`) on a running VM.
+- Rewiring the existing host-to-pod wiring on a running VM.
+
+## Non Goals
+
+- Migrating between incompatible CNI types
+- Handling failed migration
+
+## Definition of Users
+
+- Cluster admins
+- VM owners
+
+## User Stories
+
+- As a Cluster Admin, I should be able to re-assign VMs to a new VLAN ID, without having to schedule maintenance windows for reboots or downtimes.
+- As a Cluster Admin I want to immediately move a suspicious VM from the production network to a restricted network without shutting it down, so that I can cut off lateral movement and monitor traffic without alerting the attacker and while preserving the VM’s live RAM state for forensic analysis.
+- As a Cluster Admin I want to migrate VMs from a Network Attachment Definition (NAD) backed by "Physical Uplink A" to a NAD backed by "Physical Uplink B" without service interruption, so that I can perform firmware upgrades or hardware replacements on the specific ToR switch associated with Uplink A without causing downtime for the tenants.
+- As a VM Owner of a Service provider I want to promote a tenant VM from a bandwidth-restricted network profile to an unconstrained network profile instantly, so that I can apply purchased service upgrades immediately without forcing the customer to experience a disruptive reboot which would degrade their user experience.
+- As a Cluster Admin I do not want to lose my VMs in case of a failed migration
```

Comment:
Given that the current behavior already ensures the VM continues running on the source node in case a migration fails, could you please clarify what specific gap or edge case this story is addressing?